composer.json

```diff
@@ -17,10 +17,11 @@
     }
   ],
   "require": {
-    "php": ">=7.2",
+    "php": ">=7.1",
     "facebook/webdriver": "^1.5",
     "symfony/browser-kit": "^4.0",
-    "symfony/process": "^4.0"
+    "symfony/process": "^4.0",
+    "symfony/polyfill-php72": "^1.9"
```

Comment:
should be before `symfony/process` (Composer would resort the packages the next time anyway thanks to the settings below)